### ISSUE TITLE
error on importing plink data

### ISSUE THREAD

**Issue by inti** [limix/pandas-plink#8]
I am getting this error 

```In [2]: from pandas_plink import read_plink
(
In [3]: (bim, fam, G) = read_plink('Results/variants.filt_v2.0_imputed')
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-3-d18b061fd1ff> in <module>()
----> 1 (bim, fam, G) = read_plink('Results/variants.filt_v2.0_imputed')

/Users/inti/DATA/miniconda/envs/ngs/lib/python2.7/site-packages/pandas_plink/read.pyc in read_plink(file_prefix, verbose)
    103     provide a single view of the files.
    104     """
--> 105     from dask.array import concatenate
    106
    107     file_prefixes = glob(file_prefix)

/Users/inti/DATA/miniconda/envs/ngs/lib/python2.7/site-packages/dask/array/__init__.py in <module>()
      2
      3 from ..utils import ignoring
----> 4 from .core import (Array, block, concatenate, stack, from_array, store,
      5                    map_blocks, atop, to_hdf5, to_npy_stack, from_npy_stack,
      6                    from_delayed, asarray, asanyarray,

/Users/inti/DATA/miniconda/envs/ngs/lib/python2.7/site-packages/dask/array/core.py in <module>()
     19
     20 try:
---> 21     from cytoolz import (partition, concat, join, first,
     22                          groupby, valmap, accumulate, assoc)
     23     from cytoolz.curried import filter, pluck

/Users/inti/DATA/miniconda/envs/ngs/lib/python2.7/site-packages/cytoolz/__init__.py in <module>()
     16
     17 # Aliases
---> 18 comp = compose
     19
     20 # Always-curried functions

NameError: name 'compose' is not defined```

Any ideas why

**Comment by horta** [comment]
Hi @inti , yes. That is a problem with cytooz. Try to install/update it: conda install cytoolz or conda update cytools (or pip if you are not using conda)

**Comment by horta** [comment]
I've just fixed the problem in the latest version: 1.2.26. Thanks a lot for the feedback!
